example/integration_test/user_li_for_purpose_test.dart

```diff
@@ -18,6 +17,10 @@ void main() {
   final resetBtnFinder = find.byKey(Key("reset"));
   final getUserLegitimateInterestStatusForPurposeBtnFinder = find.byKey(Key("getUserLegitimateInterestStatusForPurpose"));
 
+  // Messages
+  final notReadyMessage = "Native message: Failed: 'Didomi SDK is not ready. Use the onReady callback to access this method.'.";
```

Comment:
now that I see this message is being used in many tests, how about extracting it to a new file/class?